src/v2/Components/SaleDetailTimer.tsx

```diff
@@ -0,0 +1,101 @@
+import { SaleDetailTimer_sale } from "v2/__generated__/SaleDetailTimer_sale.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import * as React from "react"
+import { Flex, Text } from "@artsy/palette"
+import { useTimer } from "v2/Utils/Hooks/useTimer"
+
+export interface SaleDetailTimerProps {
+  sale: SaleDetailTimer_sale
+}
+
+function relativeDaysUntil(parsedDays, suffix) {
```

Comment:
Let's move this helper lower down in the file. It's more customary for the exported objects near the top of the file to be the actual React component.